### PR TITLE
Added test cases to verify ka-GE locale support

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [ ] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Added test cases to verify `ka-GE`(Georgian-Georgia) locale.
 * LocaleInfo, Localematch, ScritInfo, String, Date/Time Format, Month/Weekday Translation, Duration, Date range, NumberFormat, and ListFmt test cases are added.

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/test/date/testWeekdayTranslation.js [hunk 1]
```diff
@@ -11151,6 +11151,62 @@ module.exports.testWeekdayTranslation = {
...
+        }
+        test.equal(value[0], "კვირა");
+        test.equal(value[1], "ორშაბათი");
+        test.equal(value[2], "სამშაბათი");
+        test.equal(value[3], "ოთხშაბათი");
+        test.equal(value[4], "ხუთშაბათი");
+        test.equal(value[5], "პარასკევი");
+        test.equal(value[6], "შაბათი");
+
+        fmt = new DateFmt({locale:"ka-GE", date:"w", length: "long", useNative:false, timezone:"local"})
```
Comment (ehoogerbeets) [inline js/test/date/testWeekdayTranslation.js]: This is testing 4 different things. There should be 4 unit tests: Full, Long, Medium, and Short

File: js/test/date/testcalendar.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ module.exports.testcalendar = {
         "es-CL","es-CO","es-CR","es-DO","es-EC","es-ES","es-GQ","es-GT","es-HN","es-MX","es-NI","es-PA","es-PE","es-PH","es-PR",
         "es-PY","es-SV","es-US","es-UY","es-VE","fi-FI","fr-BE","fr-BF","fr-BJ","fr-CA","fr-CD","fr-CF","fr-CG","fr-CH","fr-CI",
         "fr-CM","fr-GQ","fr-DJ","fr-DZ","fr-FR","fr-GA","fr-GN","fr-LB","fr-LU","fr-ML","fr-RW","fr-SN","fr-TG","ga-IE","gu-IN",
-        "he-IL","hi-IN","hi-SG","hr-HR","hr-ME","hu-HU","id-ID","is-IS","it-CH","it-IT","ja-JP","kk-Cyrl-KZ","km-KH","kn-IN",
+        "he-IL","hi-IN","hi-SG","hr-HR","hr-ME","hu-HU","id-ID","is-IS","it-CH","it-IT","ja-JP","ka-GE","kk-Cyrl-KZ","km-KH","kn-IN",
         "ko-KR","ku-Arab-IQ","lt-LT","lv-LV","mk-MK","ml-IN","mn-Cyrl-MN","mr-IN","ms-MY","ms-SG","nb-NO","nl-BE","nl-NL","pa-IN",
```
Comment (ehoogerbeets) [inline js/test/date/testcalendar.js]: What about ka-IR?

File: js/test/daterange/testdatefmtrange_ka_GE.js [hunk 3]
```diff
@@ -0,0 +1,693 @@
+/*
+ * testdatefmtrange_ka_GE.js - test the date range formatter object Georgian/Georgia
+ * 
+ * Copyright © 2020, JEDLSoft
+ *
+ * Licensed unenr the Apache License, Version 2.0 (the "License");
+ * you may not use tens file except in compliance with the License.
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_ka_GE.js]: Same typos as the one you fixed yesterday in other files

File: js/test/daterange/testdatefmtrange_ka_GE.js [hunk 4]
```diff
@@ -0,0 +1,693 @@
...
+}
+if (typeof(DateRngFmt) === "undefined") {
+    var DateRngFmt = require("../../lib/DateRngFmt.js");
+}
+
+if (typeof(ilib) === "undefined") {
+    var ilib = require("../../lib/ilib.js");
+}
+
+module.exports.testdatefmtrange_en_GB = {
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_ka_GE.js]: testdatefmtrange_ka_GE

File: js/test/daterange/testdatefmtrange_ka_GE.js [hunk 5]
```diff
@@ -0,0 +1,693 @@
...
+    setUp: function(callback) {
+        ilib.clearCache();
+        callback();
+    },
+
+    testDateRngFmtGBRangeInDayShort: function(test) {
+        test.expect(2);
+        var fmt = new DateRngFmt({locale: "en-GB", length: "short"});
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_ka_GE.js]: s/en-GB/ka-GE/g

There are probably a few bugs here.

File: js/test/daterange/testdatefmtrange_ka_GE.js [hunk 6]
```diff
@@ -0,0 +1,693 @@
...
+    var ilib = require("../../lib/ilib.js");
+}
+
+module.exports.testdatefmtrange_en_GB = {
+    setUp: function(callback) {
+        ilib.clearCache();
+        callback();
+    },
+
+    testDateRngFmtGBRangeInDayShort: function(test) {
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_ka_GE.js]: s/testDateRngFmtGB/testDateRngFmtGE/g

File: js/test/daterange/testdatefmtrange_ka_GE.js [hunk 7]
```diff
@@ -0,0 +1,693 @@
...
+        var end = new GregorianDate({
+            year: 2011,
+            month: 12,
+            day: 31,
+            hour: 14,
+            minute: 30,
+            second: 0,
+            millisecond: 0
+        });
+        test.equal(fmt.format(start, end), "31 Dec 2011, 13:45 – 14:30");
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_ka_GE.js]: Month names need translations

File: js/test/root/testlocalematch.js [hunk 8]
```diff
@@ -1040,6 +1040,41 @@ module.exports.testlocalematch = {
...
+            locale: "GE"
+        });
+        test.ok(typeof(lm) !== "undefined");
+        var locale = lm.getLikelyLocale();
+        test.ok(typeof(locale) !== "undefined");
+        test.equal(locale.getSpec(), "ka-Geor-GE");
+        test.done();
+    },
+
+    testLocaleMatcherGetLikelyLocaleByLocaleCode61: function(test) {
```
Comment (ehoogerbeets) [inline js/test/root/testlocalematch.js]: Need unique test function name

File: js/test/strings-ext/testSuite.html [hunk 9]
```diff
@@ -67,7 +68,7 @@
     <script src="testlistfmt_tr.js"></script>
     <script src="testlistfmt_uk_UA.js"></script>
     <script src="testlistfmt_ur.js"></script>
-    <script src="testlistfmt_zh.js"></script -->
```
Comment (ehoogerbeets) [inline js/test/strings-ext/testSuite.html]: Ooof, good catch!

File: js/test/date/testWeekdayTranslation.js (hunk 1 above)
Comment (gouniLee) [reply js/test/date/testWeekdayTranslation.js]: Regarding the format,
```
full, long --> MMMM
medium -->  MMM
short -->  MM
```
full and long are the same, and short will be written the number. That's why only two length test cases are written.

File: js/test/daterange/testdatefmtrange_ka_GE.js (hunk 4 above)
Comment (gouniLee) [reply js/test/daterange/testdatefmtrange_ka_GE.js]: I think I missed to update this datefmtrange_ka_GE tese file all. I'll update it include fixing typos.

File: js/test/date/testcalendar.js (hunk 2 above)
Comment (gouniLee) [reply js/test/date/testcalendar.js]: I will add it to the list to have the same locale list with `locales.unittest` property

Comment (gouniLee) [reply js/test/date/testcalendar.js]: Oh.. I misunderstood, I thought the comments from `build.xml` file. I'll add a calendar testcase regarding `ka-IR`.

File: js/test/date/testWeekdayTranslation.js (hunk 1 above)
Comment (ehoogerbeets) [reply js/test/date/testWeekdayTranslation.js]: What I meant is to break that 1 big unit tests into 4 separate ones, so if any of them fail, then it is easier to figure out what went wrong.

Comment (gouniLee) [reply js/test/date/testWeekdayTranslation.js]: oh.. sorry, ;(  I should've read carefully and checked the correct file. following your comment, I re-updated it. Thank you

**Pull request merged**